plugins/mcreator-core/jsontriggers/entity_hurt_player.json

```diff
@@ -0,0 +1,52 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "amount_l",
+      "check": "Number"
+    },
+    {
+      "type": "input_value",
+      "name": "amount_h",
+      "check": "Number"
+    },
+    {
+      "type": "field_input",
```

Comment:
Use the datalist instead. See the following file for how to use the argument (it's an argument we add, it's not a Blockly's one) https://github.com/MCreator/MCreator/blob/d02e808413442aafa37dea6a833795cb0e519b9d/plugins/mcreator-core/aitasks/watch_closest_entity.json#L9

Reply:
When using the Datalist, it gets written in the JSON file as EnityZombie in stead of minecraft:zombie

And the advancement JSON doesn't understand EntityZombie, so i did try this and this change was intentional.

Reply:
As Goldorion said, this is not correct. The reason why it uses class names is because of wrong mapping table being used. 

See my comment https://github.com/MCreator/MCreator/pull/4772#discussion_r1569182679

Reply:
Read Klemen's answer https://github.com/MCreator/MCreator/pull/4772#discussion_r1569182679

Reply:
I will update it immediately!